charts/datadog/values.yaml

```diff
@@ -600,6 +600,23 @@ datadog:
         # datadog.securityAgent.runtime.network.enabled -- Set to true to enable the collection of CWS network events
         enabled: false
 
+      activityDump:
+        # datadog.securityAgent.runtime.activityDump.enabled -- Set to true to enable the collection of CWS activity dumps
+        enabled: false
+
+        # datadog.securityAgent.runtime.activityDump.tracedCgroupsCount
+        tracedCgroupsCount: 3
+
+        # datadog.securityAgent.runtime.activityDump.cgroupDumpTimeout
+        cgroupDumpTimeout: 20
+
+        # datadog.securityAgent.runtime.activityDump.cgroupWaitListSize
+        cgroupWaitListSize: 0
+
+        pathMerge:
+          # datadog.securityAgent.runtime.activityDump.pathMerge.enabled
```

Comment:
Let's add a small explanation of the field in each doc string. Let's also mention the unit of `cgroupDumpTimeout`

Reply:
You are right, should be updated in the last commit